pages/Schedules.tsx

```diff
@@ -0,0 +1,156 @@
+import { Page } from '@/components/Page/Page'
+import React from 'react'
+
+export default function Schedules() {
+  return (
+    <Page>
+      <section className="flex flex-col text-center w-full items-center">
+        <h1>Schedule</h1>
+        <h3>Schedule is in PST (Pacific Standard Time)</h3>
+        <h2>Saturday, April 2</h2>
+        <div className="flex flex-row gap-12 px-12 my-6">
```

Comment:
make sure your divs have class names that describe what the div does
ex: `schedule-wrapper`